microsoft/validation/yt8m_inference_json_formatter.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2018 Microsoft All Rights Reserved.
+#
+# Active Learning Team
+# ==============================================================================
+
+r"""A simple helper script that formats Audioset Inference CSV files to a norma-
+lized consumable json to be used in model inference result comparisons. The scr-
+pt expects a directory of csv files processed my the Youtube-8m Frame Level mod-
+els and outputs a directory of respected json files converted by this script.
+Usage:
+  # Run a directory of csv files as the input.
+  # Specify a targeted output directory for the json files.
+
+  $ python yt8m_inference_json_formatter.py --csv_files /path/to/csv/files \
+                                    --json_files /path/to/targed/json/file/directory \
+
+  # Future code will include: Model Metrics, Threshold for Audioset Label Confidence
+"""
+from __future__ import print_function
+
+import os
+import csv
+import glob
+import json
+import pandas as pd
+import tensorflow as tf
+from collections import deque
+
+flags = tf.app.flags

```

Comment:
Don't use TF for passing argument flags since we don't actually use TF anywhere in this script. Use Pythons build-in [`argparser`](https://docs.python.org/3/library/argparse.html)

Look here for example:
https://github.com/evanlouie/fox-audio/blob/master/microsoft/white_noise_padding.py#L334

Reply:
Was using TensorFlow because would potentially need to directly call TF api to get model details for Model metrics in future features. What do you think?